src/index.html

```diff
@@ -1,19 +1,181 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width,
-          user-scalable=no,
-          initial-scale=1.0,
-          maximum-scale=1.0,
-          minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Document</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-
-<script type="text/javascript" src="./main.js"></script>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width,
+            user-scalable=no,
+            initial-scale=1.0,
+            maximum-scale=1.0,
+            minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>layout_catalog</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <section class="body-wrapper">
+
+      <header class="header-of-site">
+        <a href="#" class="logo-inside"><img class="logo-in-header" src="images/logo.png" alt="logo"></a>
+        <nav class="nav-links">
+          <ul class="list-of-links">
+            <li class="li-item"><a class="link Apple is-active" href="#">apple</a></li>
```

Comment:
Что значит класс `Apple`? Классы не называются с большой буквы, зачем он здесь?
```suggestion
            <li class="nav-item"><a class="nav-link is-active" href="#">apple</a></li>
```